.github/workflows/cppbuild-master.yml

```diff
@@ -3,7 +3,10 @@ on:
     branches:
       - master
   schedule:
-    - cron: 0 10 * * 1
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]> <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 18:00:00 UTC (Monday at 10:00:00 PST)
+    - cron: '0 18 * * 1'
```

Comment:
Nice, everyone forgets crons format. Cron is widely used but everyone works from examples so it's great to have the example here.